app/src/main/java/com/example/mywatchlist/adapters/ItemAdapter.kt

```diff
@@ -0,0 +1,37 @@
+package com.example.mywatchlist.adapters
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.example.mywatchlist.R
+import com.example.mywatchlist.databinding.FragmentMoviesBinding
+import dagger.hilt.android.AndroidEntryPoint
+import javax.inject.Inject
+
+
+class ItemAdapterForMovies: RecyclerView.Adapter<ItemAdapterForMovies.ItemViewHolder>() {
```

Comment:
Class should be renamed to MoviesAdapter. ItemAdapterForMovies is a terrible name.